web/src/engine/osk/src/banner/suggestionBanner.ts

```diff
@@ -753,6 +749,9 @@ export class SuggestionBanner extends Banner {
       if(suggestions.length > i) {
         const suggestion = suggestions[i];
         d.update(suggestion, optionFormat);
+        if(this.predictionContext.selected == suggestion) {
+          d.highlight(true);
+        }
```

Comment:
If we instead replace the if-condition with `suggestion?.autoHighlight`, we can keep the "autocorrect hint" around in a cleaner way.  Any interaction with the banner would instantly (and permanently) deselect it this way.